Rohit/PostgresAssignment/controller/class.ts

```diff
@@ -0,0 +1,53 @@
+import { Context } from "vm";
+
+const cls = require('../sql/class');
+
+interface classesinfo {
+    cls_id: number;
+    st_id: number;
+  }
+  
+  
+exports.classData = async (ctx: Context) => {
+    try {
+      let [rows]: Array<{ rows: classesinfo }> = [];
+      rows = await cls.get_classes();
+  
+      ctx.response.status = 200;
+      ctx.response.type = "application/json";
+      ctx.body = rows.rows;
+    } catch (err) {
+      ctx.response.status = 500;
+      ctx.response.type = "text/html";
+      ctx.body = "internal server error";
+      return;
+    }
+  };
+  
+  exports.add_student_in_classes_table = async (ctx: Context) => {
+    try {
+      let req: classesinfo = ctx.request.body;
+      if (
```

Comment:
for validation we can use https://joi.dev/api/?v=17.4.0, check this out and try to add it in few places